context/app/static/js/components/Search/SearchWrapper.jsx

```diff
@@ -184,14 +184,19 @@ SearchWrapper.propTypes = {
       type: PropTypes.oneOf([
         // Expand as needed; Starting small to catch typos.
         'RefinementListFilter',
+        'RangeFilter',
       ]).isRequired,
       props: PropTypes.exact({
+        // This is the union of two distinct prop lists.
         id: PropTypes.string.isRequired,
         title: PropTypes.string.isRequired,
         field: PropTypes.string.isRequired,
-        operator: PropTypes.string.isRequired,
-        size: PropTypes.number.isRequired,
+        operator: PropTypes.string,
```

Comment:
Do you need `is.Required` with `exact()`? I believe you'll get a warning for a missing property either way.

Reply:
I'm going to redo this to separate the types. FWIW, I don't think `exact` implies `is.Required`: Here's the example from the docs:
```
  // An object with warnings on extra properties
  optionalObjectWithStrictShape: PropTypes.exact({
    optionalProperty: PropTypes.string,
    requiredProperty: PropTypes.number.isRequired
  }),
```
👉 `exact` catches _extra_ props, not missing ones.